### ISSUE TITLE
Add an option to stop Bakery Preview switch to Bakery

### ISSUE THREAD

**Issue by Zurigan** [guycalledfrank/bakery-issues#52]
On hitting the 'Open Preview' button in Bakery Preview the tab gets switched to Bakery ... I can see why this might be useful but it would be nice to have an option to stop this happening!

**Comment by guycalledfrank** [comment]
Yeah, it's just a side-effect of instantiating ftRenderLightmap class which is both the lightmapper logic and the editor window. Annoys me as well sometimes. Will see if there's an easy workaround.
